test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -64,13 +65,13 @@ def test_jsonapi_renderer_not_registered
           'attributes' => {
             'name' => 'Johnny Rico'
           },
-          'type' => 'users'
+        'type' => 'authors'
         }
       }
       payload = '{"data": {"attributes": {"name": "Johnny Rico"}, "type": "authors"}}'
       headers = { 'CONTENT_TYPE' => 'application/vnd.api+json' }
       post '/render_with_jsonapi_renderer', params: payload, headers: headers
-      assert expected, response.body
+      assert_equal 500, response.status
```

Comment:
looks good. 

can you add an assertion for the empty body?

 If you know what the exception actually, was, would love for you to paste it and stack trace here.